src/styles/variables.ts

```diff
@@ -202,13 +204,16 @@ export default {
     oldDotWireframeIconHeight: 143.28,
     sectionIllustrationHeight: 220,
     photoUploadPopoverWidth: 335,
+    onboardingModalWidth: 500,
+    welcomeVideoDelay: 500,
 
     // The height of the empty list is 14px (2px for borders and 12px for vertical padding)
     // This is calculated based on the values specified in the 'getGoogleListViewStyle' function of the 'StyleUtils' utility
     googleEmptyListViewHeight: 14,
     hoverDimValue: 1,
     pressDimValue: 0.8,
     qrShareHorizontalPadding: 32,
+    purposeMenuIconSize: 52,
```

Comment:
Can we do 48 instead? I think we typically use 48 across the app for these smaller icons (see the header of workspace editor pages), cc @Expensify/design

Reply:
We originally used `56px` here before we started that work, but given we've since added the app bar header variation with `48px`, I'm happy to align to that. I've updated the Figma to reflect this as well.

Reply:
I've updated this value to use `48px` for the purpose modal icons.

Reply:
Thanks!